assets/js/modules/analytics-4/components/dashboard/DashboardAllTrafficWidgetGA4/UserCountGraph.js

```diff
@@ -116,7 +117,7 @@ export default function UserCountGraph( props ) {
 
 	const [ , ...ticks ] = chartData.slice( 1 ).map( ( [ date ] ) => date );
 
-	const chartOptions = { ...UserCountGraph.chartOptions };
+	const chartOptions = cloneDeep( UserCountGraph.chartOptions );
```

Comment:
Rest spread only produces a shallow copy which would result in nested properties being mutated when assigned below. This didn't seem to cause a noticeable breakage but is a bug.